src/jquery.toast.js

```diff
@@ -312,6 +312,10 @@ if ( typeof Object.create !== 'function' ) {
             this.prepareOptions(options, this.options);
             this.setup();
             this.bindToast();
+        },
+        
+        close: function() {
+            this._toastEl.find('.close-jq-toast-single').click();
```

Comment:
I would rather suggest to move the body of `click` to a separate function call and call that function.